### PR TITLE
Add roslyn 15.9 -> CLI 2.1.5xx trigger

### PR DESCRIPTION
**Pull request by nguerrera**
@jaredpar @dotnet/roslyn-infrastructure @livarcocc

### CODE REVIEW

**Comment by nguerrera** [comment]
cc @mmitche @MattGal

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1559,7 +1560,8 @@
           "GITHUB_PULL_REQUEST_NOTIFICATIONS": "dotnet/dotnet-cli",
           "GITHUB_UPSTREAM_BRANCH": "release/2.1.5xx",
           "ASPNET_VERSION_FRAGMENT": "dotnet/aspnet/release/2.1",
-          "CORESETUP_VERSION_FRAGMENT": "dotnet/core-setup/release/2.1"
+          "CORESETUP_VERSION_FRAGMENT": "dotnet/core-setup/release/2.1",          
+          "ROSLYN_VERSION_FRAGMENT": "dotnet/roslyn/dev15.9"
```
[the remark this answers is not in the document]
Comment (nguerrera) [reply Maestro/subscriptions.json]: We should wait for it to show up before merging then. There's a PR out to roslyn to set the label

[the remark this answers is not in the document]
Comment (nguerrera) [reply Maestro/subscriptions.json]: https://github.com/dotnet/roslyn/pull/30768

[the remark this answers is not in the document]
Comment (mmitche) [reply Maestro/subscriptions.json]: @dagood I think for CLI, it actually just updates the dependencis with some "missing info" elements.

[the remark this answers is not in the document]
Comment (dagood) [reply Maestro/subscriptions.json]: My thinking is the [`BuildInfo.Get` call](https://github.com/dotnet/cli/blob/release/2.1.5xx/build_projects/update-dependencies/Program.cs#L79) would [hit a 404 and fail](https://github.com/dotnet/buildtools/blob/17f28536524248603108c56a16b5b16dcb5a87ff/src/Microsoft.DotNet.VersionTools/BuildInfo.cs#L205).

The ["DEPENDENCY * NOT FOUND"](https://github.com/dotnet/buildtools/blob/17f28536524248603108c56a16b5b16dcb5a87ff/src/Microsoft.DotNet.VersionTools/Dependencies/BuildOutput/FileRegexPackageUpdater.cs#L28) should just be when a dependency updater was added, but no buildinfo contains the package id it's looking for.

[the remark this answers is not in the document]
Comment (nguerrera) [reply Maestro/subscriptions.json]: @dagood 

https://github.com/dotnet/versions/blob/master/build-info/dotnet/roslyn/dev15.9/Latest_Packages.txt is up now.

Can you merge this?

**Comment by dagood** [comment]
@nguerrera Conflicts need fixing, then I'll merge.

**Comment by nguerrera** [comment]
Conflicts resolved.

**Pull request merged**